src/status_im/ui/screens/profile/events.cljs

```diff
@@ -145,28 +145,43 @@
           (accounts-events/account-update {:name         cleaned-name
                                            :last-updated now})))))
 
+(defn check-status-change
+  "Takes map containing old status and the updated one and if the status has changed
+  returns the effects neccessary for status broadcasting"
+  [fx {:keys [old-status status]}]
+  (if (and (not= status nil)
```

Comment:
Maybe `(not (nil? status))` or simply `status` ?

Reply:
Function name is confusing. I would expect a function called `check-..` to return a boolean.

Reply:
That was the exact name of the removed event. But you are right, `status-change` looks much better. TBH, in clj(s) I only expect it to return something boolean-ish if it ends with `?`

Reply:
I think it's common idiom in clj/cljs land to consider non-nil value to be truthy, so I agree with simple `(if status ...)`

Reply:
not just common - idiomatic! changed